deploy/helm/scf/templates/implicit_vars.yaml

```diff
@@ -11,3 +12,17 @@ metadata:
 type: Opaque
 stringData:
   value: {{ .Values.system_domain | quote }}
+---
```

Comment:
It would be better to handle the `deployment_name` in a separate PR, as we need to change other places.

Reply:
E.g. https://github.com/SUSE/scf/blob/v2-develop/deploy/helm/scf/templates/bosh_deployment.yaml#L6

Reply:
Pushed a commit to fix all the remaining places with `deployment_name`.